content/general/versions/v1.1/push.textile

```diff
@@ -18,7 +18,7 @@ We no longer support "Google Cloud Messaging":https://developers.google.com/clou
 
 h3(#download). Downloading a client library with push support
 
-The following Ably client library SDKs provide support for activation and receiving of native push notifications:
+The following Ably client library SDKs provide support for activating and receiving push notifications:
```

Comment:
Ably Client Library SDKs

Reply:
> Looks good! Just a few suggestions for consistency.

Thanks! I'm resolving all the "Ably Client Library SDK" updates as I think they're out of scope for this PR and an issue already exists to address them at once.